templates/app/offline-worker.js

```diff
@@ -0,0 +1,126 @@
+/* Any copyright is dedicated to the Public Domain.
```

Comment:
I wonder if this should live at the top level of the directory tree, since it's a template, but unlike the other files in templates/, this one doesn't get copied to a developer's app directory when they bootstrap an app. Eh, it probably doesn't matter.